Backend/app/dashboard/rider-dashboard/src/Domain/Action/RiderPlatform/Management/Customer.hs

```diff
@@ -0,0 +1,31 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-# OPTIONS_GHC -Wno-unused-imports #-}
+
+module Domain.Action.RiderPlatform.Management.Customer (getCustomerList) where
+
+import qualified API.Types.RiderPlatform.Management.Customer
+import qualified Dashboard.Common
+import qualified "lib-dashboard" Domain.Types.Merchant
+import qualified "lib-dashboard" Environment
+import EulerHS.Prelude
+import qualified Kernel.Prelude
+import qualified Kernel.Types.Beckn.Context
+import qualified Kernel.Types.Id
+import Kernel.Utils.Common
+import qualified RiderPlatformClient.RiderApp.Operations
+import Storage.Beam.CommonInstances ()
+import Tools.Auth.Api
+import Tools.Auth.Merchant
+
+getCustomerList :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> ApiTokenInfo -> Kernel.Prelude.Maybe Kernel.Prelude.Int -> Kernel.Prelude.Maybe Kernel.Prelude.Int -> Kernel.Prelude.Maybe Kernel.Prelude.Bool -> Kernel.Prelude.Maybe Kernel.Prelude.Bool -> Kernel.Prelude.Maybe Kernel.Prelude.Text -> Kernel.Prelude.Maybe (Kernel.Types.Id.Id Dashboard.Common.Customer) -> Environment.Flow API.Types.RiderPlatform.Management.Customer.CustomerListRes)
+getCustomerList merchantShortId opCity apiTokenInfo limit offset enabled blocked phone personId = do
+  checkedMerchantId <- merchantCityAccessCheck merchantShortId apiTokenInfo.merchant.shortId opCity apiTokenInfo.city
+  RiderPlatformClient.RiderApp.Operations.callRiderAppOperations checkedMerchantId opCity (.customerDSL.getCustomerList) limit offset enabled blocked phone personId
+
+--- << AUTOGENERATED Check this code, update export list and remove comment >> ---
+
+deleteCustomerDelete :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> ApiTokenInfo -> Environment.Flow Kernel.Types.APISuccess.APISuccess)
```

Comment:
This domain handler was autogenerated.
But nevertheless it is located in `src` folder, not `src-read-only`, so you are able to edit this file as you need.

1. Check that logic here is the same as in old handler https://github.com/nammayatri/nammayatri/blob/main/Backend/app/dashboard/rider-dashboard/src/API/RiderPlatform/Customer.hs#L112. If something changed you can edit manually
2. Remove comment --- << AUTOGENERATED Check this code, update export list and remove comment >> ---
3. Add `deleteCustomerDelete` to module export list
4.  Check build